osh/hub/settings_local.py

```diff
@@ -25,6 +25,7 @@
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'openscanhub',
         'USER': 'openscanhub',
+        # Use osh.hub.other.settings_util.get_secret to load the secret from file.
         'PASSWORD': 'velryba',
```

Comment:
Could we use something like this?
```py
       'PASSWORD': get_secret('db_password', SECRET_KEY_DIR) or 'velryba',
```

This way a user could protect the development database with a password without preventing `settings_local.py` from being updated in the future.

Reply:
Addressed in 1cc549a8530937a6c8ee8971ae174aac21b6b93a.